core/sup_tree_core/version_upgrade_task_queue.ex

```diff
@@ -70,4 +83,12 @@ defmodule AntikytheraCore.VersionUpgradeTaskQueue do
   defun core_updated() :: :ok do
     :ok = GenServer.cast(__MODULE__, :core_updated)
   end
+
+  defun enable_version_upgrade() :: :ok do
+    :ok = GenServer.cast(__MODULE__, :enable_version_upgrade)
+  end
+
+  defun disable_version_upgrade() :: :ok do
+    :ok = GenServer.cast(__MODULE__, :disable_version_upgrade)
```

Comment:
- It's an issue about existing code, but `:ok = ` isn't necessary as `GenServer.cast/2` always returns `:ok`.
Let's remove them (4 lines).
- Names of these new functions are a bit redundant given that the module name contains "version" and "upgrade".
`enable/0` and `disable/0` should suffice.

Reply:
fixed in 2139515